application/common/config/local.test.php

```diff
@@ -1,6 +1,43 @@
 <?php
 
 return [
+    'params' => [
+        'password' => [
+            'minLength' => [
+                'value' => 10,
+                'phpRegex' => '',
```

Comment:
Since the Password model's rules will be trying to use these, should we go ahead and include the phpRegex values?